src/ripple/rpc/handlers/AccountInfo.cpp

```diff
@@ -137,7 +137,8 @@ doAccountInfo(RPC::JsonContext& context)
         // The document states that signer_lists is a bool, however
         // assigning any string value works. Do not allow this.
         // This check is for api Version 2 onwards only
-        if (!params[jss::signer_lists].isBool() && context.apiVersion > 1)
+        if (params.isMember(jss::signer_lists) &&
```

Comment:
You don't have to explicitly check if `jss::signer_lists` is a member of `params`, because otherwise a `NULL` member is returned as a default. (Source: https://github.com/XRPLF/rippled/blob/aded4a7a925a5822522e6bbbb4adfd3fec39257b/src/ripple/json/json_value.h#L341)

I'm not saying that what you did is wrong, I am making sure that the existing code is not incorrect.

Does it make a difference that you are using `1u` instead of `1`, given that the left-hand-side `context.apiVersion` is an `unsigned int`?

Reply:
Would "1" not do a type conversion from signed to unsigned ? I think "1u" avoids that extra step. No ?